<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>652</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>20</horstretch>
    <verstretch>20</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>652</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SDL Log Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/icon.png</normaloff>../images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="4">
       <widget class="QToolButton" name="toolButton_caseSensetive">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
selection-background-color: rgb(193, 125, 17);
selection-color: rgb(0, 0, 0);
color: rgb(155, 144, 146);
</string>
        </property>
        <property name="text">
         <string>Aa</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButton_findPrev">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(155, 144, 146);</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>16</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::UpArrow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="button_hide_all">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(155, 144, 146);</string>
        </property>
        <property name="text">
         <string>Hide opened</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_search_count">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(155, 144, 146);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>0 of 0</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="button_Find">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(155, 144, 146);</string>
        </property>
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="button_add_to_notes">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(155, 144, 146);</string>
        </property>
        <property name="text">
         <string>To Notes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton_findNext">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(155, 144, 146);</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>16</height>
         </size>
        </property>
        <property name="arrowType">
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="toolButton_RegularExpression">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(155, 144, 146);</string>
        </property>
        <property name="text">
         <string>*/</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="search_input_field">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
selection-background-color: rgb(193, 125, 17);
selection-color: rgb(0, 0, 0);
color: rgb(155, 144, 146);
</string>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>10</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 30, 30);
selection-background-color: rgb(50, 125, 17);
selection-color: rgb(0, 0, 0);
color: rgb(155, 144, 146);</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTabWidget" name="main_tabWidget_">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(37, 37, 38);</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_files_">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(37, 37, 38);</string>
         </property>
         <attribute name="title">
          <string>Files</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeView" name="filesView">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(30, 30, 30);
selection-background-color: rgb(193, 125, 17);
selection-color: rgb(0, 0, 0);
color: rgb(155, 144, 146);
</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_filter_">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(37, 37, 38);</string>
         </property>
         <attribute name="title">
          <string>Filter</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QPushButton" name="button_apply_filter">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="serch_input_for_filter">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(30, 30, 30);
selection-background-color: rgb(193, 125, 17);
selection-color: rgb(0, 0, 0);
color: rgb(155, 144, 146);
</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QTreeWidget" name="tree_winget_filter">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(30, 30, 30);
selection-background-color: rgb(193, 125, 17);
selection-color: rgb(0, 0, 0);
color: rgb(155, 144, 146);</string>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="toolButton_check_uncheck_all">
            <property name="text">
             <string>...</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_colums_">
         <attribute name="title">
          <string>Colums</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(30, 30, 30);
selection-background-color: rgb(193, 125, 17);
selection-color: rgb(0, 0, 0);
color: rgb(155, 144, 146);</string>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QTableView" name="main_table_widget_">
        <property name="font">
         <font>
          <family>Ubuntu</family>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);
selection-background-color: rgb(193, 125, 17);
selection-color: rgb(0, 0, 0);
color: rgb(155, 144, 146);
</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
       </widget>
      </widget>
      <widget class="QTabWidget" name="search_tabWidget_">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(30, 30, 30);</string>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <widget class="QWidget" name="tab_search_">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);</string>
        </property>
        <attribute name="title">
         <string>Search</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTableView" name="search_table_widget_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(155, 144, 146);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_notes_">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30, 30, 30);</string>
        </property>
        <attribute name="title">
         <string>Notes</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTableView" name="notes_table_widget_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(155, 144, 146);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionConfig"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
